src/core/control/LatexController.cpp

```diff
@@ -114,8 +114,10 @@ void LatexController::findSelectedTexElement() {
     if (this->selectedElem) {
         // this will get the position of the Latex properly
         EditSelection* theSelection = control->getWindow()->getXournal()->getSelection();
-        this->posx = theSelection->getXOnView();
-        this->posy = theSelection->getYOnView();
+        // when latex is edited by double-clicking on it the selection is updated to the double clicked latex element
+        // therefore we can assume there's exactly one element in the selection
+        this->posx = theSelection->getElements()[0]->getX();
+        this->posy = theSelection->getElements()[0]->getY();
```

Comment:
A slightly more elegant solution (in my opinion) to achieve the same thing would be:
```suggestion
        xoj::util::Rectangle<double> rect = theSelection->getSnappedBounds();
        this->posx = rect.x;
        this->posy = rect.y;
```

Reply:
I like the alternative. Thanks a lot!